lib/auth/register.go

```diff
@@ -242,6 +243,11 @@ func Register(params RegisterParams) (*proto.Certs, error) {
 		if err != nil {
 			return nil, trace.Wrap(err)
 		}
+	case types.JoinMethodSpacelift:
```

Comment:
v14 uses if else if :cry:

Reply:
Oh ouch - surprised this didn't conflict when cherry-picking

Reply:
[7e47013](https://github.com/gravitational/teleport/pull/34779/commits/7e47013646a008e9855b8d3ec7a7749c1211db70)